code/modules/awaymissions/corpse.dm

```diff
@@ -21,6 +21,10 @@
 	density = 1
 	anchored = 1
 
+/obj/effect/mob_spawn/New()
+	if(!mob_gender)
+		mob_gender = pick(MALE, FEMALE)
+
```

Comment:
That's what I meant by "become Rust":
http://www.cbc.ca/news/technology/rust-avatar-gender-race-assigned-1.3535949
Things like gender are better be under player control.